datasets/libsvm.py

```diff
@@ -1,40 +1,33 @@
-from benchopt import BaseDataset
-
-from benchopt import safe_import_context
-
+from benchopt import BaseDataset, safe_import_context
 
 with safe_import_context() as import_ctx:
     from libsvmdata import fetch_libsvm
-    from sklearn.preprocessing import StandardScaler
 
 
 class Dataset(BaseDataset):
 
     name = "libsvm"
 
     parameters = {
-        'dataset': ["bodyfat", "leukemia", "rcv1.binary", "YearPredictionMSD"],
+        "dataset": ["bodyfat", "YearPredictionMSD"],
     }
 
-    install_cmd = 'conda'
-    requirements = ['pip:git+https://github.com/mathurinm/libsvmdata@main']
+    install_cmd = "conda"
+    requirements = ["pip:git+https://github.com/mathurinm/libsvmdata@main"]
     references = [
         "C. Chang and CJ. Lin, "
-        "'ACM transactions on intelligent systems and technology (TIST)', "
-        "Acm New York, USA vol 2 (2011)."
+        "LIBSVM: a library for support vector machines,"
+        "ACM transactions on intelligent systems and technology (TIST), 2(3), 1-27, (2011)."
     ]
 
+
     def __init__(self, dataset="bodyfat"):
         self.dataset = dataset
+        self.X, self.y = None, None
```

Comment:
calls are cached hence there should not be a need for that, @tomMoral right ?

Reply:
Yes, avoid storing `X, y` in the dataset object, we cache the full return from `get_data` in the object to avoid multiple loading.